plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -72,6 +72,14 @@
       - This option is completely ignored if using a version of Ansible greater than C(2.9.x).
```

Comment:
Probably something similar here:
```suggestion
      - This option is completely ignored if using a version of Ansible greater than C(2.9.x).
      - Note that this option will be removed without any further deprecation warning once support
        for Ansible 2.9 is removed from this module.
```